### ISSUE TITLE
WormHoly Cannot get NSMutableURLRequest request.

### ISSUE THREAD

**Issue by TimberTang** [pmusolino/Wormholy#56]
that's my code 
```
NSMutableURLRequest *request = [requestSerializer requestWithMethod:[obj methodString] URLString:[NSString stringWithFormat:@"%@%@", [obj apiUrl]] parameters:params error:nil];
    
    NSURLSessionDataTask *dataTask = [self.manager dataTaskWithRequest:request uploadProgress:^(NSProgress * _Nonnull uploadProgress) {
        
    } downloadProgress:^(NSProgress * _Nonnull downloadProgress) {
        
    } completionHandler:^(NSURLResponse * _Nonnull response, id  _Nullable responseObject,
```

breakpoint not run here in wormholy

```
override init(request: URLRequest, cachedResponse: CachedURLResponse?, client: URLProtocolClient?) {
        super.init(request: request, cachedResponse: cachedResponse, client: client)
        
        if session == nil {
            session = URLSession(configuration: .default, delegate: self, delegateQueue: nil)
        }
    }
```

**Comment by TimberTang** [comment]
In swift 4 it is no problem !
but, i am upgreate to swift 5, from [#51](https://github.com/pmusolino/Wormholy/pull/52)
glade to have a answer!

**Comment by pmusolino** [comment]
Hi there. Are you using the fork of @iiiCeBlink or the last version of Wormholy?

**Comment by TimberTang** [comment]
> Hi there. Are you using the fork of @iiiCeBlink or the last version of Wormholy?

yeah, i am using the lastly version~

**Comment by pmusolino** [comment]
I released a new (big) version of Wormholy (1.6.5) with some improvements and fixes. https://github.com/pmusolino/Wormholy/releases/tag/1.6.5 Can you test it and let me know if you are still encountering this issue?
